database/src/main/postgres/runs/V1.5.11__get_partitioning_measures.sql

```diff
@@ -33,29 +33,48 @@ $$
 -- Returns:
 --      measure_name        - Name of the measure
 --      measured_columns    - Array of columns associated with the measure
+--      status              - Status code
+--      status_text         - Status message
+
+-- Status codes:
+--      10 - Record not found for the given partitioning
+--      11 - OK
+--      41 - Partitioning not found
 --
 -------------------------------------------------------------------------------
 
 DECLARE
     _fk_partitioning                    BIGINT;
+    _measure_name                       TEXT;
+    _measured_columns                   TEXT[];
 BEGIN
     _fk_partitioning = runs._get_id_partitioning(i_partitioning);
 
     IF _fk_partitioning IS NULL THEN
-        measure_name := 'Partitioning not found';
-        measured_columns := '{}';
+        measure_name := NULL;
+        measured_columns := NULL;
         status := 41;
         status_text := 'The partitioning does not exist.';
-        RETURN;
+        RETURN NEXT;
     END IF;
 
-    status = 11;
-    status_text = 'OK';
-
-    RETURN QUERY
-    SELECT md.measure_name, md.measured_columns, status, status_text
+    SELECT md.measure_name, md.measured_columns
```

Comment:
The original was correct. This way you would return only the first measure.
(The other option would be to cycle over the result of the `SELECT` query and do a `RETURN NEXT` for each row. Very ineffective.)